packages/eslint-config/configs/jest/rules/jest-dom.mjs

```diff
@@ -1,10 +0,0 @@
-/** @type {import('eslint').Linter.Config} */
-export const index = {
-  'jest-dom/prefer-checked': 'error',
```

Comment:
Let's use the recommended config as a base. If we want to override something, we can do it and provide a comment explaining why the override is necessary.

I think that's cleaner than manually defining every single rule like we did with jest.